OutOfSchool/OutOfSchool.WebApi.Tests/Services/BlockedProviderParentServiceTests.cs

```diff
@@ -171,25 +173,43 @@ public async Task Block_WhenDtoIsAlreadyBlocked_ShouldReturnFailed()
     public async Task Unblock_WhenDtoAndUserIdIsValid_ShouldReturnSuccess()
     {
         // Arrange
-        var parent = ParentGenerator.Generate();
-        parent.Id = unblockDto.ParentId;
+        var parent = ParentGenerator.Generate().WithUserId(userId);
+        parent.Id = blockDto.ParentId;
+        unblockEntity.Parent = parent;
+        var unblockedParentUserId = Guid.Parse(parent.UserId);
+        var additionalData = new Dictionary<string, string>()
+        {
+            { ProviderIdKey, unblockEntity.ProviderId.ToString() },
+            { ProviderFullTitleKey, unblockEntity.Provider.FullTitle },
+            { ProviderShortTitleKey, unblockEntity.Provider.ShortTitle },
+        };
+
         var blockedParents = new List<BlockedProviderParent>
         {
             unblockEntity,
         };
 
         blockedProviderParentRepositoryMock
-            .Setup(x => x.GetByFilter(It.IsAny<Expression<Func<BlockedProviderParent, bool>>>(), It.IsAny<string>()))
-            .ReturnsAsync(blockedParents);
+            .Setup(x => x.GetBlockedProviderParentEntities(It.IsAny<Guid>(), It.IsAny<Guid>()))
+            .Returns(blockedParents.AsQueryable().BuildMock());
         blockedProviderParentRepositoryMock
             .Setup(x => x.UnBlock(It.IsAny<BlockedProviderParent>()))
             .ReturnsAsync(unblockEntity);
-        parentRepositoryMock.Setup(x => x.GetById(parent.Id)).ReturnsAsync(parent);
 
         // Act
         var result = await service.Unblock(unblockDto, userId).ConfigureAwait(false);
 
         // Assert
+        notificationServiceMock
```

Comment:
(Test readability) Please use at least `blockedProviderParentRepositoryMock.VerifyAll()` in assert section, to ensure, that no excessive `mock.Setup()` is done (force you to clean up your test code) and all expected `Setup`s were called.
Similar in tests below

Reply:
I've added `blockedProviderParentRepositoryMock.VerifyAll()` and cleaned tests code. Thank you!